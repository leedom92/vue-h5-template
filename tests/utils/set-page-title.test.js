import setPageTitle from '@/utils/set-page-title'

describe('setPageTitle', () => {
  it('has custom title', () => {
    expect(setPageTitle('title')).toBe('title - Vue H5 Template').toMatchSnapshot()
  })

  it('should be Vue H5 Template', () => {
    expect(setPageTitle('')).toBe('Vue H5 Template').toMatchSnapshot()
  })
})
