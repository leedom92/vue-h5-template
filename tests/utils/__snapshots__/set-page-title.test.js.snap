// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`setPageTitle > has custom title 1`] = `"title - Vue H5 Template"`;

exports[`setPageTitle > should be Vue H5 Template 1`] = `"Vue H5 Template"`;
